contracts/round/RoundFactory.sol

```diff
@@ -21,9 +23,11 @@ import "../utils/MetaPtr.sol";
  * @dev This contract is Ownable thus supports ownership transfership
  *
  */
-contract RoundFactory is IRoundFactory, OwnableUpgradeable {
+contract RoundFactory is IRoundFactory, OwnableUpgradeable, AccessControlEnumerableUpgradeable {
   string public constant VERSION = "0.2.0";
 
+  bytes32 public constant REGISTRY_ROLE = keccak256("REGISTRY");
```

Comment:
^ ended up creating the registries as a role to allow support for multiple registries (when we look at cross-chain). We could build the permissions system on our own but need to figure that out